gobblin-cluster/src/main/java/org/apache/gobblin/cluster/GobblinHelixJobScheduler.java

```diff
@@ -148,11 +148,11 @@ public StandardMetrics getStandardMetrics() {
     private final ContextAwareCounter numJobsCommitted;
     private final ContextAwareCounter numJobsFailed;
     private final ContextAwareCounter numJobsCancelled;
```

Comment:
Do we use these metrics anywhere? They seem to occupy "good names" of the new guages